### ISSUE TITLE
VC: log next action time

### ISSUE THREAD

**Issue by arnetheduck** [status-im/nim-beacon-chain#3684]
Similar to the beacon node, each validator client instance should log the time of its next action such that an auspicious time for its restart can be picked.

```
INF 2022-05-31 15:46:00.052+02:00 Slot end                                   topics="beacnde" slot=3124730 nextActionWait=23s947ms964us877ns nextAttestationSlot=3124732 nextProposalSlot=-1 head=0bbfd9a5:3124729
```

**Comment by arnetheduck** [comment]
Just noticed this is logged in `Slot start` - however, the times are negative and growing, suggesting something is not quite right :)

```
INF 2022-05-31 15:50:12.001+02:00 Slot start                                 slot=3124751 attestationIn=-2m56s0ns blockIn=<unknown> delay=1ms395us75ns
INF 2022-05-31 15:50:24.004+02:00 Slot start                                 slot=3124752 attestationIn=-3m8s0ns blockIn=<unknown> delay=4ms404us105ns
INF 2022-05-31 15:50:36.001+02:00 Slot start                                 slot=3124753 attestationIn=-3m20s0ns blockIn=<unknown> delay=1ms492us689ns
```

**Comment by cheatfate** [comment]
This time values was positive, looks like regression.

**Comment by arnetheduck** [comment]
fixed in #3840
